R/families.R

```diff
@@ -197,6 +219,7 @@ brmsfamily <- function(family, link = NULL, link_sigma = "log",
                        link_bias = "logit", link_xi = "log1p",
                        link_alpha = "identity",
                        link_quantile = "logit",
+                       link_u = "log",
```

Comment:
I understand. I saw sigma, but I thought I would stay on the same side as `sigma = def_scale_prior(x)` in `def_par_prior()`. 

Ideally, I would like to have a parameter with `"gamma(0.01, 0.01)"` in `def_par_prior()` if link is `"identity"` and `"student_t(3, 0, 2.5)"` otherwise. For example, I tried with `kappa` and things go a bit off due to the `def_par_prior()` specification (I think),

Such specifications are present for "shape" and "phi", but both names are, unfortunately, quite loaded in a beta regression context, and I'd rather avoid them to avoid conflicts in notation. 

Is it OK to keep the extra parameter for now (perhaps with another greek letter name, say `epsilon` or a specific one like `exceedance` (or `u` as is in our paper), and then revisit in brms 3.0 (that'd be a great feature; I looked for it, when I was putting `xbetax` together :))? Or perhaps there is another way?

I will wait for your input before I continue, as, I think, the name of the parameter is important both for the codebase and the documentation. 

Many thanks,
Ioannis

Reply:
You can already achieve family-specific priors for dpars via the `prior` slot in the `.family_name` function (quite a new feature). See `.family_negbinomial` in `family-lists.R` as an example,. With this feature, could you perhaps use the existing `sigma` or `kappa`?

Reply:
Cool! Now implemented and seems to work as before - commit [641fbd7](https://github.com/paul-buerkner/brms/pull/1698/commits/641fbd726fbaecf27770ac978896bf32f8775379)